### DISCUSSION TITLE
List paired devices with 1.17.0

### DISCUSSION THREAD

**Discussion by traugott** [Koenkk/zigbee2mqtt#5712] [General]
In the pre-1.17.0-Versions there was the topic "zigbee2mqtt/bridge/config/devices/get" to get all connected/paired devices. In the 1.17.0-api i cannot find such a function anymore. The only thing i could find were the "zigbee2mqtt/bridge/devices" topic. But this topic only publishes the devices if one is leaving/joining.
For discovery the previous method (sending a message with empty payload to "zigbee2mqtt/bridge/config/devices/get") was very handy. Is there an (till now) undocumented feature for getting all paired devices ?

**Comment by Koenkk** [comment]
This is a command from the old api (deprecated but still works, docs can still be found here: https://github.com/Koenkk/zigbee2mqtt.io/blob/ead922ee141546ccce079430a7acce67c982c99b/docs/information/mqtt_topics_and_message_structure.md)

Note that `zigbee2mqtt/bridge/devices` is a retained MQTT topic meaning you will always get the latest when subscribing to it, therefore it's not needed to manually request it.
